src/mopi/run.py

```diff
@@ -1,17 +1,18 @@
 from typing import List, Tuple
 
-from constants import Const
+from mopi.plugins import WandbConfig, WandbPlugin, OutputAnalyserPlugin
+from mopi.runner.runner import Runner
+from mopi.type import Experiment, StagingConfig, StagingNames
+from mopi.runner.utils import overwrite_preprocessing_configs_
 
-from plugins import WandbConfig, WandbPlugin, OutputAnalyserPlugin
-from runner.runner import Runner
-from type import Experiment, StagingConfig, StagingNames, SourceTypes
-from runner.utils import overwrite_preprocessing_configs_
-from utils.json import dump_str
+from mopi.blocks.io import export_pipeline
 
 
 def run(
     experiments: List[Experiment],
     staging_config: StagingConfig,
+    pure_inference: bool = False,
```

Comment:
A boolean to run pure_inference -> This runs a selected lifecycles in ``Runner.run()`` not the entire pipeline.